src/util.js

```diff
@@ -1,10 +1,35 @@
 'use strict';
+function getId() {
+  const match = window.location.href.match(/https:\/\/script\.google\.com(.*?)\/home\/projects\/([^/]*)\//);
+  if (!match) return null;
+  return match[2];
+}
+
+function getRepo() {
+  return context.bindRepo[getId()];
+}
+
+function getBranch() {
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/leonhartX/gas-github/blob/083dcd77ddb6869d2b7dc2cdf2f7452c410d4901/src/util.js#L12"></span>‘getBranch’ is defined but never used.</td>
    </tr>
  </tbody>
</table>